### PR TITLE
Added /var/lib/mysql subvolume with NoCoW for older package versions of MariaDB/MySQL

### PR DESCRIPTION
**Pull request by shundhammer**

### CODE REVIEW

File: src/modules/Storage.rb [hunk 1]
```diff
@@ -5021,6 +5021,7 @@ def AddSubvolRoot(part)
         "var/lib/libvirt/images",
         "var/lib/mailman",
         "var/lib/mariadb",
+        "var/lib/mysql",
```
Comment (jreidinger) [inline src/modules/Storage.rb]: so both are valid now?

**Comment by jreidinger** [comment]
LGTM

File: src/modules/Storage.rb (hunk 1 above)
Comment (shundhammer) [inline src/modules/Storage.rb]: Right. See also explanations in the bug.

Comment (sysrich) [inline src/modules/Storage.rb]: var/lib/mysql is used by openSUSE and SLE's mariadb packages, so the addition is really required

var/lib/mariadb isn't used by any package I can find, so it's kind of useless

But, someday that might change (there's been talk by debian for example to start using /var/lib/mariadb as newer versions are not compatable with mysql) so keeping it around doesn't cost anything

Comment (shundhammer) [inline src/modules/Storage.rb]: /var/lib/mariadb was explicitly demanded in that underlying feature request, so who are we to simply remove it... ;-)

**Pull request merged**